charts/alfresco-connector-ms365/templates/deployment-connector-ms365.yaml

```diff
@@ -23,11 +23,12 @@ spec:
       labels:
         {{- include "alfresco-connector-ms365.selectorLabels" . | nindent 8 }}
     spec:
-    {{- include "component-pod-security-context" .Values | indent 4 }}
-    {{- if .Values.nodeSelector }}
-      nodeSelector: {{- .Values.nodeSelector | toYaml | nindent 8 }}
-    {{- end }}
-      {{- include "alfresco-content-services.imagePullSecrets" . | indent 6 }}
+      serviceAccountName: {{ include "alfresco-connector-ms365.serviceAccountName" . }}
+      {{- include "alfresco-common.component-pod-security-context" .Values | indent 4 }}
+      {{- if .Values.nodeSelector }}
+        nodeSelector: {{- .Values.nodeSelector | toYaml | nindent 8 }}
+      {{- end }}
```

Comment:
move it to the bottom of the deployment and add affinity/tolerations like we did in the recent PRs